src/slack/message/index.js

```diff
@@ -11,12 +17,25 @@ const getSection = (text) => ({
   },
 });
 
+/**
+ * Get text for reminders or new errand
+ *
+ * @param {object} options The options containing the reminder.
+ * @returns {string} Text based on the reminder.
+ */
 const getText = (options) => {
   return options.reminder
     ? "Reminder for a previous request"
     : "A new errand has been added";
 };
 
+/**
+ * Format heading section for slack
+ *
+ * @param {string} text The message to format.
+ * @param {boolean} reminderBoolean Whether this message is a reminder or not.
```

Comment:
Is there a JSDoc convention for object and property notation? 

There's really only one param here `options` with 2 properties `.text` (string) and `.reminder` (boolean).

Reply:
Yep absolutely. I think I missed this after rebasing master into my branch. Should be resolved now.